paasta_tools/docker_wrapper.py

```diff
@@ -295,6 +355,43 @@ def append_cpuset_args(argv, env_args):
     return argv
 
 
+def get_spark_memory_limit(node_memory_mbytes: int) -> int:
+    # by default, spark adds an overhead of 10% of the executor memory, with a
+    # minimum of 384mb
+    memory_overhead: int = max(384, int(0.1 * node_memory_mbytes))
```

Comment:
if we go with this approach, we should refactor the numbers here into named constants